pkgs/misc/vscode-extensions/default.nix

```
@@ -714,40 +745,69 @@ let
           version = "1.11.3";
           sha256 = "1smzsgcrkhghbnpy51gp28kh74l7y4s2m8pfxabb4ffb751254j0";
         };
-        meta = {
-          license = lib.licenses.mit;
-        };
+        meta = { license = lib.licenses.mit; };
       };
 
       xaver.clang-format = buildVscodeMarketplaceExtension {
         mktplcRef = {
           name = "clang-format";
           publisher = "xaver";
           version = "1.9.0";
-          sha256 = "abd0ef9176eff864f278c548c944032b8f4d8ec97d9ac6e7383d60c92e258c2f";
+          sha256 =
+            "abd0ef9176eff864f278c548c944032b8f4d8ec97d9ac6e7383d60c92e258c2f";
```

```suggestion
          sha256 = "abd0ef9176eff864f278c548c944032b8f4d8ec97d9ac6e7383d60c92e258c2f";
```

nix-fmt is great but 80 line length is not great.